src/net/net.c

```diff
@@ -95,6 +95,48 @@ int net_dst_source_addr_get(struct sa *dst, struct sa *ip)
 }
 
 
+/**
+ * Check if connection from source to destination is possible
+ *
+ * @param src Source IP address
+ * @param dst Destination IP address
+ *
+ * @return 0 if success, otherwise errorcode
+ */
+int net_check_conn(const struct sa *src, const struct sa *dst)
```

Comment:
Can you add a test case for `net_check_conn`? Something similar to: https://github.com/baresip/retest/commit/b9d03176436834c79e93774336526c7eb7d317c5